db/seeds/source_types.yml

```diff
@@ -4,6 +4,7 @@ amazon:
   :schema:
     :authentication:
     - :type: access_key_secret_key
+      :is_superkey?: true
```

Comment:
Oh, the `?` would not be much friendly for javascript code. :sweat_smile:  I know it's a Ruby standard to use it for booleans, but maybe we can make an exception here. :smile:

Reply:
@lindgrenj6 thoughts on this?

Reply:
Yep updated, my bad I immediately put `?` at the end of booleans!